pkg/vsphere/toolbox/service_test.go

```diff
@@ -273,10 +276,17 @@ func TestServiceRunESX(t *testing.T) {
 
 	service := NewService(in, out)
 
+	ping := sync.NewCond(new(sync.Mutex))
+
+	service.RegisterHandler("ping", func(b []byte) ([]byte, error) {
+		go ping.Broadcast()
```

Comment:
Since the Wait() call blocks the calling goroutine, but this will be called from the toolbox.Service's goroutine not the test's - so it isn't needed, I'll remove that.